language/German/strings.po

```diff
@@ -1787,102 +1821,102 @@ msgstr "Weitere Bedienelemente"
 #: src/frontend/mame/ui/inputmap.cpp:58
 #, c-format
 msgid "Input Assignments (%1$s)"
-msgstr ""
+msgstr "Eingabezuweisungen (%1$s)"
 
 #: src/frontend/mame/ui/inputmap.cpp:152
 msgid "Input Assignments (this system)"
-msgstr ""
+msgstr "Eingabezuweisungen (dieses System)"
 
 #: src/frontend/mame/ui/inputmap.cpp:254
 msgid "[no assignable inputs are enabled]"
-msgstr ""
+msgstr "[keine zuweisbaren Eingaben aktiviert]"
 
 #: src/frontend/mame/ui/inputmap.cpp:348
 msgid "Pressed"
-msgstr ""
+msgstr "Gedrückt"
 
 #: src/frontend/mame/ui/inputmap.cpp:418
 msgid "Invalid combination entered"
-msgstr ""
+msgstr "Ungültige Kombination eingegeben"
 
 #: src/frontend/mame/ui/inputmap.cpp:563
 #, c-format
 msgctxt "input-name"
 msgid "%1$s Analog"
-msgstr ""
+msgstr "%1$s Analog"
 
 #: src/frontend/mame/ui/inputmap.cpp:564
 #, c-format
 msgctxt "input-name"
 msgid "%1$s Analog Inc"
-msgstr ""
+msgstr "%1$s Analog Inkrement"
 
 #: src/frontend/mame/ui/inputmap.cpp:565
 #, c-format
 msgctxt "input-name"
 msgid "%1$s Analog Dec"
-msgstr ""
+msgstr "%1$s Analog Dekrement"
 
 #: src/frontend/mame/ui/inputmap.cpp:607
 #, c-format
 msgid "Press %1$s to set\n"
-msgstr ""
+msgstr "Zur Zuweisung %1$s drücken\n"
 
 #: src/frontend/mame/ui/inputmap.cpp:608
 #, c-format
 msgid "Press %1$s to append\n"
-msgstr ""
+msgstr "Zum Anhängen %1$s drücken\n"
 
 #: src/frontend/mame/ui/inputmap.cpp:609
 #, c-format
 msgid "Press %1$s to clear\n"
-msgstr ""
+msgstr "Zum Entfernen %1$s drücken\n"
 
 #: src/frontend/mame/ui/inputmap.cpp:610
 #, c-format
 msgid "Press %1$s to restore default\n"
-msgstr ""
+msgstr "%1$s drücken um den Standard zu setzen\n"
 
 #: src/frontend/mame/ui/inputdevices.cpp:32
 #, c-format
 msgctxt "menu-inputdev"
 msgid "%1$s (%2$s %3$d)"
-msgstr ""
+msgstr "%1$s (%2$s %3$d)"
 
 #: src/frontend/mame/ui/inputdevices.cpp:88
 msgctxt "menu-inputdev"
 msgid "Copy Device ID"
-msgstr ""
+msgstr "Geräte-ID kopieren"
 
 #: src/frontend/mame/ui/inputdevices.cpp:122
 msgctxt "menu-inputdev"
 msgid "Copied device ID to clipboard"
-msgstr ""
+msgstr "Geräte-ID in Zwischenablage kopiert"
 
 #: src/frontend/mame/ui/inputdevices.cpp:124
 msgctxt "menu-inputdev"
 msgid "Error copying device ID to clipboard"
-msgstr ""
+msgstr "Fehler beim Kopieren der Geräte-ID in Zwischenablage"
 
 #: src/frontend/mame/ui/inputdevices.cpp:162
 msgctxt "menu-inputdev"
 msgid "Input Devices"
-msgstr ""
+msgstr "Eingabegeräte"
 
 #: src/frontend/mame/ui/inputdevices.cpp:203
 msgctxt "menu-inputdev"
 msgid "[no input devices are enabled]"
-msgstr ""
+msgstr "[Es sind keine Eingabegeräte aktiviert]"
 
 #: src/frontend/mame/ui/inputtoggle.cpp:22
 msgctxt "menu-inputtoggle"
 msgid "Toggle Inputs"
-msgstr ""
+msgstr "Kippschalter"
```

Comment:
hmmm... "Kippschalter" is a pretty specific type of toggle switch, I don't think that helps users to know what it means. "An/Aus Einstellungen" or something like this maybe? "An/Aus Schalter"? meh, I don't know. I will probably never ever use MAME with a German language setting as it makes everything so unclear :-(

Reply:
The broader term is _Kippschalter_, as that doesn't imply On/Off, so I'd rather stick with that.

Reply:
The menu can show various multi-state inputs, not just two-state on/off.  I’m going to side with @RafTacker here.